### DISCUSSION TITLE
Post-Quantum Cryptography

### DISCUSSION THREAD

**Discussion by amlord** [panva/jose#707] [Ideas]
Hello! Following the [approval by NIST](https://csrc.nist.gov/News/2024/postquantum-cryptography-fips-approved) of 3 Post-Quantum Cryptography standards, is there a plan to support these for encryption and signing?

**Comment by panva** [comment]
@amlord when and if PQC algorithms get a) defined and [registered for use with JOSE](https://www.iana.org/assignments/jose/jose.xhtml) and b) be made available in `node:crypto` and/or Web Cryptography API

In short - this module implements only what's standardized for use with JOSE and available in the crypto runtimes it supports without dependencies.

**Reply by amlord** [reply]
ok, makes sense - i'll have a look to see if I can find any roadmaps for these
